kubernetes/deployment.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: issue-sync-prod
```

Comment:
Let's just name it `issue-sync` without the `-prod`. Information like the deployment environment is typically reserved for metadata labels.